src/Analyzer.Reports/Constants.cs

```diff
@@ -13,12 +13,12 @@ public static class Constants
         /// <summary>
         /// Tool name to be displayed in the report.
         /// </summary>
-        public const string ToolName = "Template BPA";
```

Comment:
We will need to remember to add the new values to the Defender for DevOps integration, forgetting to do that caused an incident some months ago D: I'll add a note, but an FYI for everyone